setup.cfg

```diff
@@ -1,3 +1,3 @@
 [flake8]
-ignore = E501,E226,E731,F405
+ignore = E501,E226,E731,F405,F403,F401
```

Comment:
These erros appear in __init__.py because all of the modules aren't technically used in that one file, but I want them to be imported initially. 

How do I get around this?